lib/declare_schema/model/foreign_key_definition.rb

```diff
@@ -51,8 +51,7 @@ def parent_table_name
       attr_writer :parent_table_name
 
       def to_add_statement(_new_table_name = nil, _existing_primary_key = nil)
```

Comment:
@ColinDKelley as per standup I wanted to note that these newly added args are not being used. Does it make sense to instead change this line to [fk.to_add_statement(model.table_name)](https://github.com/Invoca/declare_schema/blob/master/lib/generators/declare_schema/migration/migrator.rb#L389) to call `fk.to_add_statement` with no params since all of the data is encapsulated in the ForeignKeyDefenition object?